hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/timeline/TimestampUtils.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.client.timeline;
+
+import org.apache.hudi.common.table.HoodieTableMetaClient;
+import org.apache.hudi.common.table.timeline.HoodieTimeline;
+import org.apache.hudi.common.util.ValidationUtils;
+
+public class TimestampUtils {
+
+  public static void validateForLatestTimestamp(HoodieTableMetaClient metaClient, String instantTime) {
+    // validate that the instant for which requested is about to be created is the latest in the timeline.
+    if (!metaClient.isMetadataTable()) { // lets validate data table that timestamps are generated in monotically increasing order. 
```

Comment:
It looks like the invoker already skips the metadata table check, maybe we should eliminate this redundant check here.